chanbackup/single.go

```diff
@@ -52,6 +52,10 @@ const (
 	// SimpleTaprootVersion is a version that denotes this channel is using
 	// the musig2 based taproot commitment format.
 	SimpleTaprootVersion = 5
+
```

Comment:
Done. Commit msg is now as follows:

```
    chanbackup: add backup version for TapscriptRoot
    
    Previous to this change taproot assets channels and simple taproot channels were
    considered the same in the context of chanbackup package, since they stored the
    same data. In the following commits we are adding the data needed to produce a
    signed commitment transaction from a SCB file and in order to do that we need to
    add more fields and a custom channel gets one additional field (TapscriptRoot)
    compared to a simple taproot channel. So now we have to distinguish these kinds
    of channels in chanbackup package.
    
    See PR https://github.com/lightningnetwork/lnd/pull/8183 for more details.
```